src/sentry/static/sentry/app/views/settings/settingsIndex.tsx

```diff
@@ -243,7 +242,9 @@ class SettingsIndex extends React.Component<Props> {
                     <HomeLink to={LINKS.API}>{t('Auth Tokens')}</HomeLink>
                   </li>
                   <li>
-                    <HomeLink to={LINKS.API_APPLICATIONS}>{t('Applications')}</HomeLink>
+                    <HomeLink to={`${organizationSettingsUrl}developer-settings/`}>
+                      {t('Integrations')}
```

Comment:
@leeandher Might want to say "Your Integrations" since we are linking to the developer settings page instead of the integrations page which has all integrations.